content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Add Pentest Title to Jira Issue Summary"
+linkTitle: "Add Pentest Title to Jira Issue Summary"
+weight: 70
+---
+
+{{% pageinfo %}}
+
+This guide is designed to help [**Integration Builder**](/integrations/integrationbuilder/) users customize the summary field of 
+Jira issues created by their recipes to contain the title of the relevant pentest.
+Please follow the [Create Jira tickets for Findings guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration) before proceeding with this guide.
+
+{{% /pageinfo %}}
+
+1. Navigate to the __[Cobalt > Jira Cloud] Push pentest finding to Jira__ recipe that you set up while [following the previous guide](/integrations/integrationbuilder/how-to-guides/jira-cloud-migration).
+
+2. Open the edit view for the recipe by clicking the __Edit__ button.
+
+3. Hover your cursor above the __Monitor__ block that encapsulates the __Create issue in Jira__ action. An __Add step__ button will appear. Click on it.
```

Comment:
Steps 3)-12) could be included in a separate document about guarding remote calls with retry and error logging.

This is a great write-up, but does it imply that we will document everything (at least one per page) with this error-handling technique?

Reply:
> Steps 3)-12) could be included in a separate document about guarding remote calls with retry and error logging.

Good idea, I logged a ticket for the creation of this new guide page.
https://zombie.atlassian.net/browse/FX-4988
In the meantime, I will merge this guide with these steps so that I can send the documentation to the customer and move on to some pressing topics.

> This is a great write-up, but does it imply that we will document everything (at least one per page) with this error-handling technique?

I think in the future we can just reference the error handling guide that will be created from FX-4988